Zeeguu Reader/Zeeguu Reader/ArticleViewController.swift

```diff
@@ -523,5 +571,25 @@ class ArticleViewController: UIViewController, WKNavigationDelegate, WKScriptMes
 			Utils.pronounce(word: word, inLanguage: self.article?.feed.language)
 		}
 	}
+	
+	func like(sender: UIBarButtonItem) {
+		Utils.sendMonitoringStatusToServer("userLikesArticle", value: "1", data: ["url": self.article.url])
```

Comment:
Having an Utils packages is often a sign that  your system is not modularized well...